perf/benchmark/run_benchmark_job.sh

```diff
@@ -44,23 +45,31 @@ function setup_metrics() {
 }
 
 function collect_metrics() {
-  local GENERATE_GRAPH=$1
-  local PLOT_METRIC=$2
-  CSV_OUTPUT="$(mktemp /tmp/benchmark_XXXX.csv)"
+  # shellcheck disable=SC2155
+  export CSV_OUTPUT="$(mktemp /tmp/benchmark_XXXX.csv)"
```

Comment:
Instead of a random name can this use Local_output_dir/some meaningful name?

Reply:
It is hard to tag this name since there is only one csv file which contains all data inside. But anyway this random name is just for temp local output and the csv would be uploaded under the folder with meaningful name as benchmark_dt_sha/